### PR TITLE
[Mixtral] Add mixtral moe

### PR DESCRIPTION
**Pull request by DesmonDay**
### PR types
New features

### PR changes
Models

### Description
Add mixtral model.

### CODE REVIEW

File: llm/mixtral/lora_argument.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
+{
+    "model_name_or_path": "mistralai/Mixtral-8x7B-Instruct-v0.1",
```
Comment (lugimzzz) [inline llm/mixtral/lora_argument.json:2]: 新增一下sft的训练脚本

**Review by lugimzzz — DISMISSED** [review]

File: llm/mixtral/lora_argument.json (hunk 1 above)
Comment (DesmonDay) [reply llm/mixtral/lora_argument.json:2]: done

**Comment by wtmlon** [comment]
lgtm

**Review by lugimzzz — APPROVED** [review]
lgtm

**Pull request merged**